test/AspNetCoreModule.Test/Framework/TestWebSite.cs

```diff
@@ -308,10 +327,30 @@ public TestWebSite(IISConfigUtility.AppPoolBitness appPoolBitness, string logger
                 
                 if (InitializeTestMachine.UsePrivateAspNetCoreFile == true && IisServerType == ServerType.IISExpress)
                 {
-                    iisConfig.AddModule("AspNetCoreModule", ("%IIS_BIN%\\" + InitializeTestMachine.PrivateFileName), null);
+                    if (TestUtility.IsOSAmd64)
+                    {
+                        if (_appPoolBitness == IISConfigUtility.AppPoolBitness.enable32Bit)
+                        {
+                            iisConfig.AddModule("AspNetCoreModule", (InitializeTestMachine.IisExpressAspnetcore_X86_path), null);
+                        }
+                        else
+                        {
+                            iisConfig.AddModule("AspNetCoreModule", (InitializeTestMachine.IisExpressAspnetcore_path), null);
+                        }
+                    }
+                    else
+                    {
+                        iisConfig.AddModule("AspNetCoreModule", (InitializeTestMachine.IisExpressAspnetcore_path), null);
+                    }
+                }
+
+                string hostname = "";
```

Comment:
This is repeated twice, abstract to a new method.

Reply:
Ah, good catch!!! Fixed.